.github/workflows/python-publish.yml

```diff
@@ -22,12 +22,10 @@ jobs:
       uses: actions/checkout@v3
     - uses: actions/setup-python@v4
       with:
-        python-version: '3.10'
+        python-version: '3.11'
     - name: Install prereq
-      run: pip install wheel
-    - name: Build python package
-      run: python setup.py build
-    - name: Package python package
-      run: python setup.py sdist bdist_wheel
+      run: pip install build wheel
+    - name: Build Python package
+      run: python -m build
```

Comment:
do this provide both sdist bdist_wheel at once?

Reply:
https://pypa-build.readthedocs.io

Reply:
i saw that before, but just asking out of curiosity, do that provide same functionalities in a simpler new way?

Reply:
`build` is the PyPA default tool for doing builds without a `setup.py` but there are many other ways to do builds.  I suggest we look at the build results to see if they fit our purposes.